packaging_automation/publish_docker.py

```diff
@@ -200,7 +223,7 @@ def validate_and_extract_manual_exec_params(manual_trigger_type_param: str, tag_
 
     pipeline_trigger_type, image_type = validate_and_extract_general_parameters(args.image_type,
                                                                                 args.pipeline_trigger_type)
-    triggering_event_info, current_branch = decode_triggering_event_info(args.github_ref)
+    current_branch = get_current_branch(os.getcwd())
```

Comment:
Before I was using github_ref to get the current branch but github_ref includes tag name in case of tag push so that I used get_current_branch method to get the current branch